### ISSUE TITLE
Make it easier to deal with variables while crafting a request in builder

### ISSUE THREAD

**Issue by natanavra** [a85/POSTMan-Chrome-Extension#6797]
**Describe the bug**
Can't use `Path Variables` in the host part of the URL (need at least 3 "/" symbols for the UI to shop up), this used to work on older versions (and the UI shows up if I use an old saved request that was configured that way)

**To Reproduce**
1. In the URL input enter `https://:host`
2. Note the `Path Variables` UI section doesn't appear.
3. Changing the URL input to `https:///:host` makes the UI appear

**Expected behavior**
Path variables / Dynamic variables should be available throughout the URL

**Screenshots**
Expected (older versions and occassionally shows up for a moment when selecting a request from history): 
![Alt](https://i.imgur.com/UB4jL1Y.png)

Current (bug): 
![Alt](https://imgur.com/a/EuhAj6w.png)

**App information (please complete the following information):**
 - App Type: Native
 - Postman Version 7.2.2
 - OS: macOS High Sierra 10.13.4

**Comment by shamasis** [comment]
Hello. I’m afraid that the path variables format (one’s starting with colon,) is not intended to be supported in host part of the URL and is only available for “path”. You can use standard variables that are enclosed within double braces. 

Older versions of Postman used to show erratic behaviour and as such, has been fixed in newer versions of Postman.

**Comment by natanavra** [comment]
What's the reason not to support it in the host part?
Even if only a tiny bit of people use it... as in the mentioned examples - testing Shopify APIs requires using the shop's domain.

Using environment variables is not easily accessible to changing it at a rapid rate, unless using "Pre request script" but that moves the option to a different pane.

**Comment by shamasis** [comment]
@natanavra - you’re bang on the buck. Instead of introducing an incorrect concept, we should make it easier to deal with variables. 

@vkaegis - something to keep a track of.

**Comment by natanavra** [comment]
@shamasis Environment variables are environment, it's not something that is configured on the request level.
That's why path variables and query parameters windows were introduced.
If you change the title of "Path Variables" to "URL Variables", fits the concept and works perfectly.
Why do more work?

**Comment by shamasis** [comment]
Yep. That’s the goal... at least partially. Postman Variables support sessions and encrypted server side storage. Those features must be extended to variables and subsequently applied to entire request instead of URL. We don’t want to introduce such an invasive feature for short term only to change again.

cc @vkaegis
